cluster/libvirt-coreos/config-default.sh

```diff
@@ -19,3 +19,40 @@
 # Number of minions in the cluster
 NUM_MINIONS=${NUM_MINIONS:-3}
 export NUM_MINIONS
+
+# The IP of the master
+export MASTER_IP="192.168.10.1"
+
+export INSTANCE_PREFIX=kubernetes
+export MASTER_NAME="${INSTANCE_PREFIX}-master"
+
+# Map out the IPs, names and container subnets of each minion
+export MINION_IP_BASE="192.168.10."
+MINION_CONTAINER_SUBNET_BASE="10.10"
+MASTER_CONTAINER_NETMASK="255.255.255.0"
+MASTER_CONTAINER_ADDR="${MINION_CONTAINER_SUBNET_BASE}.0.1"
+MASTER_CONTAINER_SUBNET="${MINION_CONTAINER_SUBNET_BASE}.0.1/24"
+CONTAINER_SUBNET="${MINION_CONTAINER_SUBNET_BASE}.0.0/16"
+for ((i=0; i < NUM_MINIONS; i++)) do
+  MINION_IPS[$i]="${MINION_IP_BASE}$((i+2))"
```

Comment:
This won't work for more than 252 nodes or so. Probably don't need to worry about fixing it at this point, but maybe add a comment/TODO -- and we really need a library to centralize the IP allocation. :/